action.yml

```diff
@@ -302,6 +310,32 @@ runs:
           echo "Running PHPStan"
           vendor/bin/phpstan analyse
         fi
+        # Validate licenses of all dependencies are permissive
+        echo "Checking licenses of all dependencies"
+        composer global require madewithlove/license-checker
+        COMPOSER_GLOBAL_HOME=$(composer -q -n config --global home)
+        # A list of allowed software licesnses that are permissive - see https://spdx.org/licenses/ for a list of SPDX identifiers
+        # IMPORTANT! If this is updated also update the same variable in the "Run JS tests" step
+        SPDX_ALLOWED_DELIMITED="MIT;MIT-0;ISC;0BSD;BSD-2-Clause;BSD-3-Clause;Apache-2.0;Python-2.0;CC0-1.0;CC-BY-3.0;CC-BY-4.0;Public Domain;Unlicense"
```

Comment:
This is defined in two places as you cannot use an env variable at the parent level

We could create a new module as a central place for this variable, but I think that's overkill

Reply:
Can we put it in a file or something instead?

Reply:
Updated